cdblib/cdbdump.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+
+import argparse
```

Comment:
I used `argparse` in this new script, which does the error handling and such automatically. Probably I'll re-write `python-pure-cdbmake` to use it as well.